bezier_library/src/bezier_library.cpp

```diff
@@ -474,13 +442,10 @@ bool Bezier::cutMesh(const vtkSmartPointer<vtkPolyData> poly_data,
   maxBound[1] = poly_data->GetBounds()[3];
   maxBound[2] = poly_data->GetBounds()[5];
 
-  double center[3];
-  center[0] = poly_data->GetCenter()[0];
-  center[1] = poly_data->GetCenter()[1];
-  center[2] = poly_data->GetCenter()[2];
+  double distanceMinMax = sqrt(vtkMath::Distance2BetweenPoints(minBound,maxBound));
 
-  double distanceMin = sqrt(vtkMath::Distance2BetweenPoints(minBound, center));
-  double distanceMax = sqrt(vtkMath::Distance2BetweenPoints(maxBound, center));
+  double width = effector_diameter_ * (1 - covering_percentage_);
```

Comment:
Is it really `effector_diameter_` we want here?

I'd say we want the cutting width of the effector rather than the effector diameter in the case of grinding.